lib/roby/coordination/models/base.rb

```diff
@@ -113,12 +113,11 @@ def parse_names(suffixes)
                 return if !definition_context
 
                 # Assign names to tasks using the local variables
-                vars = definition_context.eval "local_variables"
-                values = definition_context.eval "[#{vars.map { |n| "#{n}" }.join(", ")}]"
-                vars.zip(values).each do |name, object|
+                definition_context.local_variables.each_with_object(Hash.new) do |var, h|
```

Comment:
```suggestion
                definition_context.local_variables.each do |var|
```
Is **h** used for something?

Reply:
Not anymore. Thanks